interview-questions.md

```diff
@@ -8,62 +8,63 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-  Your answer:
+  Your answer: A parameter gets passed into a function for example, and the argument is like a reference to the parameter inside the codeblock.
 
-  Researched answer:
+  Researched answer: After research, function parameters are listed in the function definition. Function arguments are the values passed to the function.
 
 
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-  Your answer:
+  Your answer: From memory, .map() takes in a value, index and the array. Of the three, only value is required. It is the same with other methods like filter.
 
-  Researched answer:
+  Researched answer: After researching to verify my answer, I verified that my answer was correct. I pulled the syntax from w3 schools to paste here. - array.map(function(currentValue, index, arr), thisValue) -
 
 
 
 3. What is the difference between map and filter?
 
-  Your answer:
+  Your answer: From memory, map loops over and "does something" to each element it touches along with returning the original array. Filter makes a "decision" on each element it touches and returns a portion or subset of an array.
 
-  Researched answer:
+  Researched answer: After researching this answer, the answers found were just like mine with a slightly different wording.
 
 
 
 4. What is the difference between a function and a method?
 
-  Your answer:
+  Your answer: From memory, a function is a rusable code machine that a developer makes to complete a given task or tasks. Methods are also built by developers but they are "built in", common use functions that devlopers have decided are used so much that essentially "javascript function presets" were made and called methods. Why capitalize words over and over when you could simply create a built in method to achieve the same thing with a single method.
```

Comment:
There is a difference between built-in methods and a method that can slightly resemble a function, for instance in a class we use methods to perform an action much like a function.  The difference, in this case, is the method belongs to an object.